orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/services/LogoutServiceTest.java

```diff
@@ -328,15 +375,20 @@ void shouldDeleteSessionFromRedisWhenNoCookieExists() {
         input.setRequestContext(contextWithSourceIp("123.123.123.123"));
 
         logoutService.handleLogout(
+                Optional.of(session),
                 Optional.empty(),
                 Optional.of(CLIENT_LOGOUT_URI),
                 Optional.of(STATE.getValue()),
                 auditUserWhenNoCookie,
                 Optional.empty(),
                 rpPairwiseId);
 
-        verify(sessionService, times(0)).deleteSessionFromRedis(SESSION_ID);
```

Comment:
Although due to the refactor this test is now testing for a session being deleted as it's name suggests, the fact that it wasn't before makes me think this was for some old logic that has now changed. The auditUserWhenNoCookie doesn't seem to trigger any specific logic?

Reply:
After looking into this it appears this test always asserted deleteSessionFromRedis was not called despite it's name. I'm removing as it's confusingly named and for the current version f the logout service doesn't test any unique logic.

Reply:
Hesitant to delete the test even though its confusingly not properly tested. Since this has gone through a refactor is it that the test hasn't been setup correctly. Seems like the test was purposefully named and should be kept and updated unless someone can confirm this logic change is fine.

Reply:
Should this test be moved into the logoutHandler as thats where the cookie logic is - testing that the `logoutService.destroySession` is still called when no cookie exists.

Reply:
@billrdunn do you have any thoughts? It looks like this was here since you created this class: https://github.com/govuk-one-login/authentication-api/pull/3647/commits/f374da8b0bf5193ae982c005e38ba94a69323adc

Reply:
Looks like there is a change in behaviour around emitting metrics though?

Reply:
As discussed yesterday, will maintain the current behaviour for now, which dosen't delete the session if the cookie is missing. I've opened a new ticket to investigate the desired behaviour.